common/fixation_entry.go

```diff
@@ -348,6 +408,47 @@ func (fs *FixationStore) PutEntry(ctx sdk.Context, index string, block uint64) {
 	fs.putEntry(ctx, entry)
 }
 
+// DelEntry deletes marks the entry for deletion so that future GetEntry will
+// not find it anymore. (Cleanup of the EntryIndex will be done when the last
+// reference is removed).
+func (fs *FixationStore) DelEntry(ctx sdk.Context, index string) error {
+	safeIndex, err := types.SanitizeIndex(index)
+	if err != nil {
+		return sdkerrors.ErrNotFound.Wrapf("invalid non-ascii index")
```

Comment:
provide additional details to the error? like the index or a more detailed description (specifically which function failed)

Reply:
this is a library function. the error clearly indicates the failure: its (single) argument was invalid.
the caller may/should add details as it sees fit.